### DISCUSSION TITLE
x86 apps

### DISCUSSION THREAD

**Discussion by zandiarash** [dotnet/dotnet-docker#5856] [Q&A]
Is there a way to run an x86 dotnet 8  WebApi which uses some x86 dlls in docker ?
I mean something like these
`FROM mcr.microsoft.com/dotnet/aspnet-x86:8.0 AS base`
`FROM mcr.microsoft.com/dotnet/sdk-x86:8.0 AS build`

**Comment by mthalman** [comment]
Official images for x86 are not provided. You can follow the guidance in https://github.com/dotnet/dotnet-docker/blob/main/documentation/scenarios/installing-dotnet.md for authoring your own Dockerfile that installs .NET. This way you can customize which architecture is used.

**Reply by mthalman** [reply]
Be aware that x86 binaries of .NET are only provided for Windows.

**Reply by zandiarash** [reply]
Actually I'm not experienced with docker
I can run my x64 web apps but I don't know what to write in my DockerFile for x86
